BaseStation/MainWindow/MainWindow.xaml.cs

```diff
@@ -50,11 +52,10 @@ public MainWindow()
 
             Settings.PropertyChanged += SettingChanged;
             SettingPanel.Settings = Settings;
-            if (!File.Exists(Directory.GetCurrentDirectory() + @"\Images\" + Settings.CurrentMapFile))
+            if (!File.Exists(Path.Combine(Directory.GetCurrentDirectory(), "Images", Settings.CurrentMapFile)))
```

Comment:
Crashes when Settings.CurrentMapFile is null (the case when any new computer runs the program) or all map data is cleared.